tests/test_core.py

```diff
@@ -306,35 +312,42 @@ def job():
         pass
 
     ha.HomeAssistant._async_add_hass_job(hass, ha.HassJob(job))
-    assert len(hass.loop.call_soon.mock_calls) == 0
-    assert len(hass.loop.create_task.mock_calls) == 0
-    assert len(hass.loop.run_in_executor.mock_calls) == 2
+    assert hass.loop.call_soon.call_count == 0
+    assert hass.loop.create_task.call_count == 0
+    assert hass.loop.run_in_executor.call_count == 1
```

Comment:
Note `hass.loop.run_in_executor` changed from `2` mock_calls to `1` call_count. That's because one of the calls isn't to `run_in_executor` directly but rather to `call().add_done_callback` which isn't counted in `call_count`.